cms/envs/common.py

```diff
@@ -175,6 +175,7 @@
 
     # for consistency in user-experience, keep the value of the following 3 settings
     # in sync with the ones in lms/envs/common.py
+    # See annotations in lms/envs/common.py for details.
     'ENABLE_DISCUSSION_SERVICE': True,
     'ENABLE_TEXTBOOK': True,
```

Comment:
Please copy your `# See annotations in lms/envs/common.py for details.` comment above this line as well.  Thank you.